CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
-## Changelog
+# Changelog
+All notable changes to this project will be documented in this file.
 
-### v0.1.0
+## [v0.2.0]
+### Added
+  - Configurable kubernetes job retries (on failure)
+### Modified
+  - Changelog format
+  - Logrus dependency path
+
+## v0.1.0 - 2018-10-08
+### Added
   - Add Authentication Headers: Email-Id and Access-Token
   - Add Job success and failure metrics
+
+[0.2.0]: https://github.com/gojektech/proctor/compare/v0.1.0...v0.2.0
```

Comment:
why this link?

Reply:
We're following a similar structure of CHANGELOG across all repos. It's inspired from this repo: https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md

Reply:
This link will take you straight to a comparison with previous tag/release